src/js/import.js

```diff
@@ -213,14 +210,9 @@
     //Default entrytype is research and default delimiter is comma.
     document.getElementById("importCheckResearch").click();
     document.getElementById("selectDelimiter").dispatchEvent(new Event('change'));
```

Comment:
Haven't seen this before, but dispatching your own events can be tricky. I don't know all the tricky cases but it's probably good to be aware that there be (some) :dragon:s here